p2p/src/lib.rs

```diff
@@ -39,34 +37,82 @@ pub mod net;
 pub mod peer;
 pub mod proto;
 
+const MAX_ACTIVE_CONNECTIONS: usize = 32;
```

Comment:
Btw, bitcoin can take up to 125, with 16 max as outbound.

Reply:
It should probably be command line option, default to whatever, but let people play with it